qa/evil-tests/src/test/resources/org/elasticsearch/common/logging/hierarchy/log4j2.properties

```diff
@@ -1,10 +1,14 @@
-status = error
-
 appender.console.type = Console
 appender.console.name = console
 appender.console.layout.type = PatternLayout
 appender.console.layout.pattern = %m%n
 
+appender.file.type = File
+appender.file.name = file
+appender.file.fileName = ${sys:es.logs}.log
+appender.file.layout.type = PatternLayout
+appender.file.layout.pattern = [%p][%l] %marker%m%n
```

Comment:
It is referred to a few lines down in `rootLogger.appenderRef.file.ref` but I don't think that appender is actually needed at all (probably just a copy-paste from another configuration when setting this test up). I'll push a commit that removes it after I'm certain that it's not needed.